config/application.rb

```diff
@@ -1,6 +1,15 @@
 require_relative 'boot'
 
-require 'rails/all'
```

Comment:
Usually, you don't need to edit application.rb configuration. Can you explain why you required only several rails components?